docs/auth/identity-resolver.md

```diff
@@ -209,55 +242,48 @@ of lower-level calls:
-        }
-
-        // This step calls the catalog to look up a user entity. You could for example
-        // replace it with a call to a different external system.
-        const { entity } = await ctx.findCatalogUser({
-          annotations: {
-            'acme.org/email': email,
-          },
-        });
+// ...
+async signInResolver({ profile: { email} }, ctx) {
+  if (!email) {
+    throw new Error('User profile contained no email');
+  }
+
+  // This step calls the catalog to look up a user entity. You could for example
+  // replace it with a call to a different external system.
+  const { entity } = await ctx.findCatalogUser({
+    annotations: {
+      'acme.org/email': email,
+    },
+  });
 
-        // In this step we extract the ownership references from the user entity using
-        // the standard logic. It uses a reference to the entity itself, as well as the
-        // target of each `memberOf` relation where the target is of the kind `Group`.
-        //
-        // If you replace the catalog lookup with something that does not return
-        // an entity you will need to replace this step as well.
-        //
-        // You might also replace it if you for example want to filter out certain groups.
-        //
-        // Note that `getDefaultOwnershipEntityRefs` only includes groups to which the
-        // user has a direct MEMBER_OF relationship. It's perfectly fine to include
-        // groups that the user is transitively part of in the claims array, but the
-        // catalog doesn't currently provide a direct way of accessing this list of
-        // groups.
-        const ownershipRefs = getDefaultOwnershipEntityRefs(entity);
-
-        // The last step is to issue the token, where we might provide more options in the future.
-        return ctx.issueToken({
-          claims: {
-            sub: stringifyEntityRef(entity),
-            ent: ownershipRefs,
-          },
-        });
-      };
-    }
-  })
-)
+  // In this step we extract the ownership references from the user entity using
+  // the standard logic. It uses a reference to the entity itself, as well as the
+  // target of each `memberOf` relation where the target is of the kind `Group`.
+  //
+  // If you replace the catalog lookup with something that does not return
+  // an entity you will need to replace this step as well.
+  //
+  // You might also replace it if you for example want to filter out certain groups.
+  //
+  // Note that `getDefaultOwnershipEntityRefs` only includes groups to which the
+  // user has a direct MEMBER_OF relationship. It's perfectly fine to include
+  // groups that the user is transitively part of in the claims array, but the
+  // catalog doesn't currently provide a direct way of accessing this list of
+  // groups.
+  const ownershipRefs = getDefaultOwnershipEntityRefs(entity);
+
+  // The last step is to issue the token, where we might provide more options in the
+  // future.
+  return ctx.issueToken({
```

Comment:
Is the auth http service being used to generate the token? I believe it could be helpful to share a link to more detailed documentation for those who want to learn how tokens are generated. I have some questions, such as whether private claims can be included in the token, for example.

Reply:
the auth resolver context link added for the above comment ^ has links to the different types involved, the docs there could be made more explicit too as a separate effort